pkg/patch/patch.go

```diff
@@ -148,3 +144,14 @@ func escapeJSONPointer(key string) string {
 	s0 := strings.ReplaceAll(key, "~", "~0")
 	return strings.ReplaceAll(s0, "/", "~1")
 }
+
+func isAppMeshCNIEnabled(meta Meta) bool {
+	if v, ok := meta.PodMetadata.Annotations[config.AppMeshCNIAnnotation]; ok {
+		return v == "enabled"
```

Comment:
So if the pod has "appmesh.k8s.aws/appmeshCNI: disabled" and "eks.amazonaws.com/fargate-profile: XXXX", AppMesh CNI will be disabled. This is intended, yes? Just making sure...

Reply:
yeah, but not sure if anyone will use that combo. I thought about throwing error but seemed unnecessary.

Reply:
I think throwing an error may be sane thing here given that the pod will not work as expected. Can you tell me what the behavior will be if mutation webhook throws error ?

Reply:
No, I think it's fine to leave it as-is. Just wanted to make sure that was intended, nothing more :)